cli/endpoints/online/managed/binary-payloads/code/single-file-to-file-score.py

```diff
@@ -0,0 +1,30 @@
+from azureml.contrib.services.aml_request import AMLRequest, rawhttp
+from azureml.contrib.services.aml_response import AMLResponse
```

Comment:
Ditto

Reply:
It doesn't seem like there's a way to get rid of these, although the user doesn't actually have to have the package installed.